drivers/sensor/tdk/icm42670/Invn/InvError.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @defgroup InvError Error code
+ *	@brief    Common error code
+ *
+ *	@ingroup EmbUtils
+ *	@{
+ */
+
+#ifndef _INV_ERROR_H_
+#define _INV_ERROR_H_
+
+/** @brief Common error code definition
+ */
+enum inv_error {
+	INV_ERROR_SUCCESS = 0,    /**< no error */
+	INV_ERROR = -1,           /**< unspecified error */
+	INV_ERROR_NIMPL = -2,     /**< function not implemented for given arguments */
+	INV_ERROR_TRANSPORT = -3, /**< error occurred at transport level */
+	INV_ERROR_TIMEOUT = -4,   /**< action did not complete in the expected time window */
+	INV_ERROR_SIZE = -5, /**< argument's size is not suitable to complete requested action */
+	INV_ERROR_OS = -6,   /**< error related to OS */
+	INV_ERROR_IO = -7,   /**< error related to IO operation */
+	INV_ERROR_MEM = -9,  /**< not enough memory to complete requested action */
+	INV_ERROR_HW = -10,  /**< error at HW level */
```

Comment:
not sure what these are? why not use `errno` codes?

Reply:
Thank you for the review. 
This InvError.h file is part of our driver package (that should move to be in hal_tdk module). I removed any use of them in icm42670.c file.